WordPress/src/main/res/layout/theme_activation_bottom_sheet.xml

```diff
@@ -0,0 +1,173 @@
+        android:layout_below="@id/info_tv"
+        android:layout_marginEnd="@dimen/margin_extra_large"
+        android:layout_marginStart="@dimen/margin_extra_large"
+        android:layout_marginTop="@dimen/margin_extra_extra_medium_large">
+
+        <ImageView
+            android:id="@+id/use_theme_check_iv"
+            android:layout_width="@dimen/theme_activation_bottom_sheet_selector_size"
+            android:layout_height="@dimen/theme_activation_bottom_sheet_selector_size"
+            android:layout_alignParentEnd="true"
+            android:layout_centerVertical="true"
+            android:contentDescription="@string/theme_activation_bottom_sheet_selected_description"
+            android:src="@drawable/ic_tick_vector_blue"
+            android:visibility="invisible" />
+
+        <TextView
+            android:id="@+id/use_theme_layout_sub_text_tv"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/use_theme_layout_main_text_tv"
+            android:text="@string/theme_activation_use_theme_layout_sub_text"
+            android:textSize="@dimen/theme_activation_bottom_sheet_text_size_small" />
+
+        <TextView
+            android:id="@+id/use_theme_layout_main_text_tv"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/theme_activation_use_theme_layout_main_text"
+            android:textColor="@color/black"
+            android:textSize="@dimen/theme_activation_bottom_sheet_text_size" />
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/theme_activation_bottom_sheet_divider_height"
+            android:layout_below="@+id/use_theme_layout_sub_text_tv"
+            android:layout_marginTop="@dimen/margin_medium"
+            android:background="@color/gray_5" />
+    </RelativeLayout>
+
+    <RelativeLayout
+        android:id="@+id/keep_current_layout_ll"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/use_theme_layout_ll"
+        android:layout_marginEnd="@dimen/margin_extra_large"
+        android:layout_marginStart="@dimen/margin_extra_large"
+        android:layout_marginTop="@dimen/margin_medium">
+
+        <ImageView
+            android:id="@+id/keep_current_check_iv"
+            android:layout_width="@dimen/theme_activation_bottom_sheet_selector_size"
+            android:layout_height="@dimen/theme_activation_bottom_sheet_selector_size"
+            android:layout_alignParentEnd="true"
+            android:layout_centerVertical="true"
+            android:contentDescription="@string/theme_activation_bottom_sheet_selected_description"
+            android:src="@drawable/ic_tick_vector_blue" />
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="30dp"
```

Comment:
We should avoid hardcoded values and uses dimensions and/or styles for this.

Reply:
Missed this one.

Reply:
This looks better :) Thank you for fixing this @kashandanish.